cmd/puppet-naginator/templates.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/nightlyone/puppetquery"
+)
+
+const nagiosPrefix = "Nagios_"
+
+var keyOrder = slice2map([]string{"host_name", "alias", "use", "parents", "service_description"})
+var badparams = slice2map([]string{"notify", "target", "ensure", "require", "before", "tag"})
```

Comment:
those are called metaparameter in puppet. Except target. I don't know target. Here's a list: https://docs.puppetlabs.com/references/latest/metaparameter.html